esx_service/vmodl/vmodl_test.py

```diff
@@ -59,12 +59,13 @@
 NEW_TENANT_DESC = "TEST_TENANT_DESCRIPTION_2"
 TENANT_PREFIX = "TEST_TENANT_"
 
-def connect_to_vcs(host = "localhost", port = 443):
+def connect_to_vcs(host="localhost", port=443):
     """
     Connect to VCS - currently utilizing VSAN mgmt service on ESX (/vsan) - and return SOAP stub
     """
 
     si = vmdk_ops.get_si()
+    #pylint: disable=no-member
```

Comment:
No, it's not commented out. This is the standard way to disable a pylint warning: http://stackoverflow.com/questions/4341746/how-do-i-disable-a-pylint-warning